extensions/azurecore/package.json

```diff
@@ -128,6 +128,105 @@
               "Microsoft Authentication Library"
             ],
             "deprecationMessage": "Warning: ADAL has been deprecated, and is scheduled to be removed in a future release. Please use MSAL (default option) instead."
+          },
+          "azure.providerSettingsJson": {
+            "type": "array",
+            "description": "%config.providerSettingsJson%",
+            "scope": "resource",
+            "items": {
+              "type": "object",
+              "properties":{
+                "name": {
```

Comment:
Please add descriptions to all properties and include public endpoints as defaults